draft-ietf-moq-requirements.md

```diff
@@ -675,6 +680,11 @@ groups and just wait for the next group when needed.
 
 Of course, having longer blocks create other issues. Realtime conferences also need to accomodate the occasional occasional late comer, or the disconnected user who want to resynchronize after a network event. This drives a need for synchronization "between access points". For example, rather than waiting for 30 seconds before connecting, the user might quickly download the "key" frames of the past 30 seconds and replay them in order to "synchronize" the video decoder.
 
+### Media Insertion and Redirection
+
+In all use cases it may be necessary for consumers to switch sources based on either triggering provided within a stream of media, or externally in particular to support the insertion of advertising or promotions, but may be useful for other requirements where an upstream encoder or producer is unable to do. Relays may also require consumers to relocate their subscriptions of media in order to facilitate this.
+
+In order to make seemless switching from one source to another and avoid stalling of playback, consumers must be able to receive media from sources concurrently. Support for common in-band signalling formats such as {{SCTE-35}} should also be supported.
```

Comment:
>  **Support** for common in-band signalling formats such as {{SCTE-35}} **should also be supported**.

is a bit redundant. I'd suggest either 

>  Common in-band signalling formats such as {{SCTE-35}} should also be supported.

or

>  Support for common in-band signalling formats such as {{SCTE-35}} should also be **included**.

Reply:
I have such a bad habit of that type of redundancy, I'll fix it thanks.